third_party/arm_compiler.BUILD

```diff
@@ -1,75 +0,0 @@
-package(default_visibility = ["//visibility:public"])
-
-filegroup(
-    name = "gcc",
-    srcs = glob(["bin/*-gcc"]),
-)
```

Comment:
The arm compiler definition and indeed all of `third_party/toolchains` already exists in the TF upstream; now that we import the TF workspace directly, we get those for free and no longer need to define them in LCE ourselves, so they can be deleted.

Reply:
Hmm or perhaps not, since CI isn't happy with it. It seemed to work locally in a docker container, but I'll investigate.

Reply:
Could it be that they are using a different compiler version?

Reply:
Yeah so it turns out that Aarch64 works but Arm32 doesn't.

It's because their definition of [compiler pieces](https://github.com/tensorflow/tensorflow/blob/22ba168c4b843c3f3a5c397f2bd45552d8477fbb/arm_compiler.BUILD#L43-L51) differs from [what we had before](https://github.com/larq/compute-engine/blob/f7f9352ba804f30e2fe1c6a9742c97699807ddf6/third_party/arm_compiler.BUILD#L43-L51) - I'm trying to work out how to work around this.

Reply:
I've opened https://github.com/tensorflow/tensorflow/pull/48479 to fix the `compiler_pieces` upstream; if they merge it quickly then we can ask for a cherry-pick onto the `r2.5` branch and hopefully use that directly. Otherwise, we'll just have to revert to registering our own cross-compiler like we previously did.

Reply:
Nice, thanks for the fix. Alternatively, we could also try their [embedded toolchains](https://github.com/tensorflow/tensorflow/tree/master/third_party/toolchains/embedded/arm-linux) which seem to do a very similar thing.

Reply:
Yeah that's a good point, those are also a newer GCC version. There must be some reason that they still have the older toolchain around, maybe to support older RPi boards? There's a comment in [rpi-newer-crosstools](https://github.com/rvagg/rpi-newer-crosstools/tree/eb68350c5c8ec1663b7fe52c742ac4271e3217c5) about OS support, so maybe it's that.

Reply:
Yes, I think the `rpi-newer-crosstools` predate the embedded toolchains which have been added in https://github.com/tensorflow/tensorflow/commit/4961f18733ca3967198393abf419e14476b4a85c to enable TFLite `pip` wheels for RPI in https://github.com/tensorflow/tensorflow/commit/ff997783433bd0975c4daab84facd22d139d555a

Reply:
I hadn't realised, but this is indeed what they recommend for RPi boards: https://www.tensorflow.org/lite/guide/build_arm#cross-compilation_for_arm_with_bazel

Reply:
And indeed using the embedded linux toolchains fixed the ARM tests :tada: